doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -227,6 +227,11 @@ nRF Desktop
 
 * Added a debug configuration enabling the `Fast Pair`_ feature on the nRF54L15 PDK with the ``nrf54l15pdk/nrf54l15/cpuapp`` board target.
 
+* Added DVFS module to the nRF Desktop application.
+  The module is responsible for switching the frequency and voltage according to the application's needs.
+  The module ensures that if there are no DVFS requests application is running on the lowest frequency and voltage.
```

Comment:
ensures that lowest DVFS frequency and voltage is used unless a computation-challenging operation is performed.